packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.test.tsx

```diff
@@ -98,7 +97,8 @@ describe('DatePicker', () => {
     const datePicker = mount(<DatePickerBase allowTextInput={true} onSelectDate={onSelectDate} />);
 
     datePicker.setState({ isDatePickerShown: true });
-    ReactTestUtils.Simulate.click(document.querySelector('.ms-DatePicker-day--today') as HTMLButtonElement);
+
+    datePicker.find('.ms-DatePicker-day--today').simulate('click');
```

Comment:
This is not working as the element you are looking its part of a different component (`Calendar`). At least locally it's not. Wait to see what Travis has to say.